homeassistant/components/zwave_js/sensor.py

```diff
@@ -394,6 +394,13 @@ def __init__(
         self._attr_force_update = True
         self._attr_name = self.generate_name(include_value_name=True)
 
+    @property
+    def options(self) -> list[str] | None:
```

Comment:
What kind of sensors use this property? Why did we move it?

Reply:
I don't recall why I moved it to be honest. It only makes sense when the sensor is of type ENUM, and I think for now, that only is possible with `ZWaveListSensor` and `ZWaveConfigParameterSensor`. The thought may have been that since it based on device class, it was safest to include in the base class for future proofing

Reply:
I think it's more clear if we keep it in the "lowest" common class in the tree where it has a function.

Reply:
I'm confusing PRs. This shouldn't have moved in this PR to begin with